Plugins/Weapon/Weapon.cpp

```diff
@@ -490,9 +550,34 @@ int32_t Weapon::GetEpicWeaponDevastatingCritical(NWNXLib::API::CNWSCreatureStats
    {
       auto w = plugin.m_EpicWeaponDevastatingCriticalMap.find(pWeapon->m_nBaseItem);
       feat =  (w == plugin.m_EpicWeaponDevastatingCriticalMap.end()) ? -1 : w->second;
+   }    
+   bFlag = feat>-1 ? pStats->HasFeat(feat) : plugin.m_GetEpicWeaponDevastatingCriticalHook->CallOriginal<int32_t>(pStats, pWeapon);
+
+   if(bFlag && !plugin.m_DCScript.empty()) 
+   {
+      NWNXLib::API::CNWSCreature         *pCreature    = pStats->m_pBaseCreature;
+      NWNXLib::API::CNWSCombatRound      *pCombatRound = pCreature->m_pcCombatRound;
+      NWNXLib::API::CNWSCombatAttackData *pAttackData  = pCombatRound->GetAttack(pCombatRound->m_nCurrentAttack);
+       
+      plugin.m_DCData.oidWeapon = pWeapon->m_idSelf;
+      plugin.m_DCData.oidTarget = pCreature->m_oidAttackTarget;
+      plugin.m_DCData.nDamage   = pAttackData->GetTotalDamage(1); 
+      plugin.m_DCData.bBypass   = false;
+      
+      NWNXLib::API::CExoString script = plugin.m_DCScript.c_str();
+
+      LOG_DEBUG("Devastating Critical Event Prev Call Script %s", plugin.m_DCScript.c_str());
+      Globals::VirtualMachine()->RunScript(&script, pCreature->m_idSelf, 1);
```

Comment:
You can just do `Utils::ExecuteScript(plugin.m_DCScript, pCreature->m_idSelf)`. Include "Utils.hpp", no need to include the VM stuff